### PR TITLE
Feat: Added Image and input components

### PR DESCRIPTION
**Pull request by LordUttam**
Added Image and Input components. 
Also refactored some code for the docs page.
Please review, thank you :)

### CODE REVIEW

File: pages/docs.css [hunk 1]
```diff
@@ -237,253 +236,253 @@ body {
...
-    line-height: 1.375rem;
-    font-weight: 600;
-    padding: .5rem 4px;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  font-size: 1rem;
+  line-height: 1.375rem;
+  font-weight: 600;
+  padding: 0.5rem 4px;
```
Comment (NeerajVodala) [inline pages/docs.css]: use either px or rem

File: pages/docs.css [hunk 2]
```diff
@@ -237,253 +236,253 @@ body {
...
+  padding: 1rem;
+  font-size: 2rem;
+  background-color: var(--primary-color);
+  color: var(--base-color);
 }
 
 .main__heading h1 {
-    margin: 0px;
-    margin-left: 1rem;
+  margin: 0px;
```
Comment (NeerajVodala) [inline pages/docs.css]: no need to mention unit

File: pages/docs.css [hunk 3]
```diff
@@ -237,253 +236,253 @@ body {
...
+  font-size: 1.5rem;
+  background-color: var(--secondary-bg-color);
+  color: var(--neutral-color);
 }
 
 .main__sub-heading h2 {
-    margin: 0px;
-    margin-left: 1rem;
+  margin: 0px;
+  margin-left: 1rem;
```
Comment (NeerajVodala) [inline pages/docs.css]: use shorthand property if possible

File: styles/inputs.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+.input__container--row {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.input {
+  padding: 0.5rem;
+  margin: 0.5rem 0 0.5rem 0;
```
Comment (NeerajVodala) [inline styles/inputs.css]: 0.5rem 0 would work

**Review by NeerajVodala — COMMENTED** [review]
Try to be consistent with the units.
Rest everything looks good to me.
All the best.

File: pages/docs.css [hunk 5]
```diff
@@ -237,253 +236,253 @@ body {
...
-    margin: 0;
-    margin-left: 0;
-    width: 20%;
-    height: 90vh;
-    position: fixed;
-    top: 20%;
-    box-shadow: var(--secondary-box-shadow);
-}
-    
+  margin: 0;
```
Comment (rishbitsnbytes) [inline pages/docs.css:255]: No need to use margin left if you are using margin. Shorthand would be best here.

File: pages/docs.css (hunk 1 above)
Comment (rishbitsnbytes) [inline pages/docs.css]: Unit consistency. If you are using rem, then use rem at all places. Change px here to rem.

**Review by rishbitsnbytes — COMMENTED** [review]
Have a lookk at the comments.

**Comment by rishbitsnbytes** [comment]
Rest looks good.

**Comment by LordUttam** [comment]
Thanks guys! I have made the suggested corrections.

**Pull request merged**